pkg/service/utils_service.go

```diff
@@ -387,3 +388,22 @@ func getClientIP(r *http.Request) (string, string) {
 	}
 	return sourceIP, xForwardedFor
 }
+
+// detectCharset detects charset for a given text in a buffer.
+func detectCharset(buffer []byte) (string, error) {
+	if len(buffer) > 32768 {
+		buffer = buffer[:32768]
+	}
+	// Create detector.
+	detector := chardet.NewTextDetector()
+	// Detect charset.
+	result, err := detector.DetectBest(buffer)
+	if err != nil {
+		return "", fmt.Errorf("error detectando charset: %w", err)
+	}
+	// If confidence is low, consider it as UTF-8.
+	if result.Confidence < 10 {
```

Comment:
where is this confidence value "10" comes from?

Reply:
Threshold suggested by module's original author. Moved to a constant